pipeline/README.md

```diff
@@ -64,3 +78,11 @@ We score each scenario model in two ways:
 ## Running the pipeline
 
 The pipeline structure is built using [DrWatson.jl](https://github.com/JuliaDynamics/DrWatson.jl) for project management of simulation parameters/settings, saved results, and figures. Compute is done with [Dagger.jl](https://github.com/JuliaParallel/Dagger.jl) for organizing parallel computation and checkpointing of results.
+
+## Running pipeline tests
+
+To run the tests for the pipeline, run the following command in the Julia REPL with the root directory as the working directory:
+
+```julia
+include("pipeline/test/runtests.jl")
```

Comment:
can't we make them be in the pipeline directory for all of these and use test?

Reply:
Not precisely sure what you mean?

Reply:
I agree that this way of testing is fairly clunky. Maybe a clever use of `@testitem`? I am conscious that these will ultimately live in different places so I don't want to intermingle the pipe tests with the `EpiAware` tests too much

Reply:
I'm just confused why you don't tell people to activate the pipeline environment and run test?

Reply:
That doesn't work the way this is set up... I can change it so that could work though.

Reply:
as `pipeline` has no environment? I don't feel strongly if there is a strong reason for the way its set up at the mo?

Reply:
once we have clarity on this its all good to go from my perspectivee

Reply:
It does have an env but I've not set up the name to have an associated `test`, just left as manual. So if you activate the env and just do 

`> test`

it chucks

> `trying to test unnamed project `

Reply:
Basically, if I'm not adding it to the test CI or the nice `TestItem` interface then I'm acting relaxed about having the tests as a manual `include(path/to/tests)`.

Obviously, mileage can vary on how relaxed I should be.

Reply:
why wouldn't we have another CI job for this that just triggers when the pipeline folder is updated?